test/runtests.jl

```diff
@@ -351,4 +351,11 @@ end
     end
 end
 
-@test success(pipeline(`$(MPI.mpiexec()) -n 2 $(Base.julia_cmd()) mpitest.jl`, stderr=stderr, stdout=stdout))
+@testset "MPI" begin
+    if Sys.iswindows()
+        @test LAMMPS.API.lammps_config_has_mpi_support() == 0
+    else
+        @test LAMMPS.API.lammps_config_has_mpi_support() != 0
+        @test success(pipeline(`$(MPI.mpiexec()) -n 2 $(Base.julia_cmd()) mpitest.jl`, stderr=stderr, stdout=stdout))
```

Comment:
We should make this test error if it is run without MPI support

Reply:
So the plan is to fix MPI for windows and until that just have our tests automatically fail on windows?
I think we should also check wheter MPI is enabled in the __init__ function and display a warning if it isn't. We should probably do the same for errors as well, as the REPL just crashes when lammps encounters an error if that isn't the case - Alltough starting from the 2024 versions errors are allways enabled